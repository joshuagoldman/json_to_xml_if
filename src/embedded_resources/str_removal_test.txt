 <friends>
  <name 9c89481e-730a-4cc3-8d2d-1fec233283d1>Alice Williams</name>
  <age e661a4c6-2f12-477f-9907-25147c8ccc6d>null</age>
  814b194a-faf5-432c-9f4c-e402a89a94d6<age_attributes e661a4c6-2f12-477f-9907-25147c8ccc6d>
   <namn>s</namn>
   <yrke>Snuspetare</yrke>
  </age_attributes>814b194a-faf5-432c-9f4c-e402a89a94d6
  4e315479-9c93-4295-9b06-d604a4bdcc76<name_attributes 9c89481e-730a-4cc3-8d2d-1fec233283d1>
   <namn>Niclas</namn>
   <yrke>Snuspetare</yrke>
  </name_attributes>4e315479-9c93-4295-9b06-d604a4bdcc76
 </friends>thread 'tests::test_parse_to_soap_xml' panicked at src/json_to_if/mod.rs:311:5:
