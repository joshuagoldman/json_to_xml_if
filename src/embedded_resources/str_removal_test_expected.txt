 <friends>
  <name 9c89481e-730a-4cc3-8d2d-1fec233283d1>Alice Williams</name>
  <age e661a4c6-2f12-477f-9907-25147c8ccc6d>null</age>
 </friends>thread 'tests::test_parse_to_soap_xml' panicked at src/json_to_if/mod.rs:311:5:
